spec/controllers/tags_controller_spec.rb

```diff
@@ -17,6 +17,41 @@
     end
   end
 
+  shared_examples "an action only fully authorized admins can access" do |authorized_roles:|
```

Comment:
To me, "fully" implies the admin has write access. How about this?
```suggestion
  shared_examples "an action only authorized admins can access" do |authorized_roles:|
```